ec2/templates/user_data_jenkins_docker.sh.tpl

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 echo ECS_CLUSTER=${jenkins_cluster_name} > /etc/ecs/ecs.config
-chown 1000:1000 /var/run/docker*
\ No newline at end of file
+chown 1000:1000 /var/run/docker*
+yum install -y amazon-cloudwatch-agent
+amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c ssm:${agent_policy_parameter_name}
```

Comment:
Missing line break

Reply:
I found this comment https://github.com/nationalarchives/tdr-terraform-modules/pull/106/files#r636863512 where I said I was going to do it around May so I think it's probably time now. I've added on in.